src/views/Pipeline.vue

```diff
@@ -117,6 +117,48 @@ export default defineComponent({
         event.target.complete();
       })
     },
+    async skipJob (job: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurance based on the temporal expression.'),
```

Comment:
I think the text is not added to en.json. Please check for other cases as well
```suggestion
          message: this.$t('Skipping will run this job at the next occurence based on the temporal expression.'),
```

Reply:
The text was already added to en.json file.